content/guides/understanding-inbox-messaging.md

```diff
@@ -0,0 +1,77 @@
+# Understanding Inbox Messaging
+
+---
+
+1. TOC
+{:toc}
+
+---
+
+## Preface
+
+BookingSync provides the ability to integrate with the Inbox on multiple different channels/platforms we support. This guide will present
+how to implement this integration.
+
+
+## Purpose
+
+The purpose of this feature is to provide a seamless integration between internal messaging systems in certain Channels and the BookingSync platform, for adminstrators to be able to handle incoming inquiries in a centralized location.
+
+
+## Entities
+
+- **Inbox Conversation**: Represents a thread/conversation and is scoped under a Channel, an Account, and a Booking.
+<br>
+- **Inbox Participant**: Represents a participant in the thread and is scoped under an Inbox Conversation. There are two types of participants
+  - Client: Represents a Traveller, managed by the Channel entirely.
+  - Host: Represents a User in the Inbox Conversation's Account, managed by the Channel and the Account. All Users should be able to view the Inbox Conversations of their corresponding Accounts. However, Only Host Inbox Participants could send messages and be notified about new messages in the Inbox Conversation.
+<br>
+- **Inbox Message**: Represents a message in the thread/conversation and is scoped and an Inbox Participant.
+
+
+## Flow
+
+When a Booking is created on the following channels: (Booking.com, Airbnb), a corresponding Inbox Conversation is created on our system, which is synchronized with the Channel's internal messaging systems.
+
+When an Inbox Conversation is created via a Channel, at least one Client Inbox Participant will be added to the conversation automatically. A Host Inbox Participant may or may not be created via the Channel automatically.
```

Comment:
`may not be created via the Channel automatically.` - good to mention that in such case, the API integration is responsible for creating a Host participant.